src/components/viewer/report-editor/ReportEditor.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { createResource, deleteResource } from 'redux-json-api';
+
+import styles from './ReportEditor.scss';
+import {
+  AnnotationAPIModel,
+  AnnotationReportAPIModel,
+  AnnotationReportAPICreateModel,
+  AnnotationReportResourceType, AnnotationResourceType,
+  Reasons,
+} from 'api/annotations';
+import { hideEditor } from '../../../store/actions';
+import Report from './Report';
+import { PPScopeClass } from '../../../class_consts';
+import Suggestion from './Suggestion';
+
+interface IReportEditorProps {
+  annotation: AnnotationAPIModel;
+  onCancel: (e) => void;
+  onSuccess: () => void;
+  createAnnotationReport: (instance: AnnotationReportAPICreateModel) => Promise<AnnotationReportAPIModel>;
+}
+
+interface IReportEditorState {
+  activeDialog: 'menu'|'report'|'suggestion';
```

Comment:
Why not use the `enum Dialogs` since it's already declared and used?

Reply:
ups, stale leftover, I'll fix it, thx